tests/unit/test_endpoint_provider.py

```diff
@@ -111,6 +115,11 @@ def ruleset_dict():
         return json.load(f)
 
 
+@pytest.fixture(scope="module")
+def ruleset_dict():
+    return _ruleset_dict()
```

Comment:
What prompted this?

Reply:
Needed a ruleset without account ID configured for a test case here
https://github.com/boto/botocore/blob/b819d815398162ea6a9603678239d3159e8ffcbf/tests/unit/test_endpoint_provider.py#L622

I could separate this out into it's own test, which would let me use it as a fixture and remove the need for this change.